### ISSUE TITLE
Can you remove the thank you message at the end of the program? I plan on using your calculator in all of my programs and I can't figure out how to remove it.

### ISSUE THREAD

**Issue by MatteoRaso** [AceLewis/my_first_calculator.py#105]

**Comment by ghost** [comment]
Any updates?

**Comment by MatteoRaso** [comment]
> Any updates?

No. I think the devs abandoned the project. Just when I thought I found a useful and compact program smh.

**Comment by sanan-fataliyev** [comment]
did you guys find any workaround?

**Comment by LZL0** [comment]
It shall remain a mystery.

**Comment by TeCoEd** [comment]
I found a work around, copy the line of code and paste it underneath, then the program runs and prints out a message, 

print("Thanks for using this calculator, goodbye :)")
print("Ignore the previous line")

User error if they decide not to ignore the instruction, code its fixed!

**Comment by oonqt** [comment]
I tried to remove it but i cant because my computer runs out of memory opening the file in the vs code app because electron does not haneedl this big file well .

@TeCoEd how much MEMORY U HAVE ? ? ?

**Comment by TeCoEd** [comment]
I have a really high spec computer set up, it has 2 bytes of RAM and a 16 byte Hard Drive, runs Minecraft at -0.0036 FPS
